slither/visitors/slithir/expression_to_slithir.py

```diff
@@ -211,7 +212,31 @@ def _post_assignement_operation(self, expression):
                 # a = b = 1;
                 set_val(expression, left)
 
+    def _attempt_constant_folding(self, expression):
+        try:
+            const_fold = ConstantFolding(expression, expression.type)
+        except (NotConstant, AttributeError):
+            const_fold = None
+            return False
+        if const_fold is not None:
```

Comment:
As a minor nitpick, we could reduce the level of indentation by returning early here:
```
if const_fold == None:
  return False
const_value = const_fold.result()
... code here ...
return True
```